apps/daimo-mobile/src/view/sheet/OwnRequestBottomSheet.tsx

```diff
@@ -74,16 +75,19 @@ export function OwnRequestBottomSheet({
 
   return (
     <View style={ss.container.padH16}>
-      <Spacer h={16} />
+      <ScreenHeader
+        title="You requested"
+        onExit={() => {
+          dispatcher.dispatch({ name: "hideBottomSheet" });
+        }}
+        hideOfflineHeader
+      />
+      <TitleAmount amount={dollarsToAmount(reqStatus.link.dollars)} />
+      <Spacer h={8} />
       <TextCenter>
-        <TextH3>Requested</TextH3>
+        <TextBodyCaps color={color.grayMid}>USDC • BASE</TextBodyCaps>
```

Comment:
wouldn't hardcode it. see SettingsScreen.tsx AccountHeader

Reply:
fixed